.coveragerc

```diff
@@ -799,7 +799,6 @@ omit =
     homeassistant/components/smarthab/cover.py
     homeassistant/components/smarthab/light.py
     homeassistant/components/sms/*
-    homeassistant/components/smtp/notify.py
```

Comment:
Coverage needs to be 95+ % per module to not lower our average. If we don't reach that, we can keep the file excluded from coverage.

Reply:
done, submitted in a new commit.